assets/ui.css

```diff
@@ -9,6 +9,10 @@
   background: transparent;
 }
 
+.respec-modal .close-button:focus() {
```

Comment:
Please remove this, we'll do this when browsers add [`:focus-visible`](https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible#Browser_compatibility) support.

Reply:
Focusing on close button by default sounds weird, but not sure what to do.